src/utilities/useSubscanHost/useSubscanHost.ts

```diff
@@ -2,7 +2,7 @@ import { useAsyncValue } from '../useAsyncValue/useAsyncValue';
 import { getEndpoint, KnownEndpoints } from '../endpoints/endpoints';
 
 const subscanHosts: Record<KnownEndpoints, string | undefined> = {
-  'wss://peregrine.kilt.io': 'https://kilt-testnet.subscan.io',
+  'wss://peregrine.kilt.io': undefined,
```

Comment:
Another option would be to just delete this line

Reply:
True. 

It was just to be consistent with:
```
  'wss://peregrine-stg.kilt.io/para': undefined,
```

Reply:
Not true.* :/

Removing the undefined lines causes this error:
```
Type '{ 'wss://spiritnet.kilt.io': string; 'wss://kilt.dotters.network': string; 'wss://kilt.ibp.network': string; }' is missing the following properties from type 'Record<"wss://kilt.ibp.network" | "wss://kilt.dotters.network" | "wss://spiritnet.kilt.io" | "wss://peregrine.kilt.io" | "wss://peregrine-stg.kilt.io/para", string | undefined>': "wss://peregrine.kilt.io", "wss://peregrine-stg.kilt.io/para"
```

Reply:
`const subscanHosts: Record<KnownEndpoints, string | undefined> = { `needs to assign to all `KnownEndpoints`.

Reply:
Oh, my types were smarter than I remember… You did the right thing for consistency, I didn’t pay attention to the other line.